src/sql/azdata.proposed.d.ts

```diff
@@ -140,6 +140,7 @@ declare module 'azdata' {
 
 	export interface ModelBuilder {
 		radioCardGroup(): ComponentBuilder<RadioCardGroupComponent>;
+		horizontalSelector(): ComponentBuilder<HorizontalSelectorComponent>;
```

Comment:
Why is this called horizontal selector? Normally the direction the cards flow is the direction you'd name it - so this would be a verticalSelector

Reply:
I know terrible name, i wanted to avoid anything that said "card" due to CSS issues.

The name defo needs to change